Compiled from "WorkerUtils.java"
public class io.undertow.util.WorkerUtils {
  private io.undertow.util.WorkerUtils();
    Code:
       0: aload_0
       1: invokespecial #1                  // Method java/lang/Object."<init>":()V
       4: return

  public static org.xnio.XnioExecutor$Key executeAfter(org.xnio.XnioIoThread, java.lang.Runnable, long, java.util.concurrent.TimeUnit);
    Code:
       0: aload_0
       1: aload_1
       2: lload_2
       3: aload         4
       5: invokevirtual #2                  // Method org/xnio/XnioIoThread.executeAfter:(Ljava/lang/Runnable;JLjava/util/concurrent/TimeUnit;)Lorg/xnio/XnioExecutor$Key;
       8: areturn
       9: astore        5
      11: aload_0
      12: invokevirtual #4                  // Method org/xnio/XnioIoThread.getWorker:()Lorg/xnio/XnioWorker;
      15: invokevirtual #5                  // Method org/xnio/XnioWorker.isShutdown:()Z
      18: ifeq          42
      21: getstatic     #6                  // Field io/undertow/UndertowLogger.ROOT_LOGGER:Lio/undertow/UndertowLogger;
      24: aload         5
      26: ldc           #7                  // String Failed to schedule task %s as worker is shutting down
      28: aload_1
      29: invokeinterface #8,  4            // InterfaceMethod io/undertow/UndertowLogger.debugf:(Ljava/lang/Throwable;Ljava/lang/String;Ljava/lang/Object;)V
      34: new           #9                  // class io/undertow/util/WorkerUtils$1
      37: dup
      38: invokespecial #10                 // Method io/undertow/util/WorkerUtils$1."<init>":()V
      41: areturn
      42: aload         5
      44: athrow
    Exception table:
       from    to  target type
           0     8     9   Class java/util/concurrent/RejectedExecutionException
}
